feature/gribi/ate_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -537,18 +587,286 @@ func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, good, bad, lb []
+	p3 := dut.Port(t, "port3")
+	p4 := dut.Port(t, "port4")
+	btrunk2 = netutil.NextAggregateInterface(t, dut)
+
+	i2 := &oc.Interface{Name: ygot.String(btrunk2)}
+	gnmi.Replace(t, dut, d.Interface(*i2.Name).Config(), configInterfaceDUT(i2, &dutPort2, dut))
+	T2 := configureBundle(t, p2.Name(), *i2.Name)
+	gnmi.Replace(t, dut, gnmi.OC().Interface(p2.Name()).Config(), T2)
+
+	btrunk3 = netutil.NextAggregateInterface(t, dut)
+	i3 := &oc.Interface{Name: ygot.String(btrunk3)}
+	gnmi.Replace(t, dut, d.Interface(*i3.Name).Config(), configInterfaceDUT(i3, &dutPort3, dut))
+	T3 := configureBundle(t, p3.Name(), *i3.Name)
+	gnmi.Replace(t, dut, gnmi.OC().Interface(p3.Name()).Config(), T3)
+
+	btrunk4 = netutil.NextAggregateInterface(t, dut)
+	i4 := &oc.Interface{Name: ygot.String(btrunk4)}
+	gnmi.Replace(t, dut, d.Interface(*i4.Name).Config(), configInterfaceDUT(i4, &dutPort4, dut))
+	T4 := configureBundle(t, p4.Name(), *i4.Name)
+	gnmi.Replace(t, dut, gnmi.OC().Interface(p4.Name()).Config(), T4)
+
+	if deviations.ExplicitInterfaceInDefaultVRF(dut) {
+		fptest.AssignToNetworkInstance(t, dut, *i2.Name, deviations.DefaultNetworkInstance(dut), 0)
+		fptest.AssignToNetworkInstance(t, dut, *i3.Name, deviations.DefaultNetworkInstance(dut), 0)
+		fptest.AssignToNetworkInstance(t, dut, *i4.Name, deviations.DefaultNetworkInstance(dut), 0)
+	}
+	staticARPWithSecondaryIP(t, dut, true, &bundleName{trunk2: btrunk2, trunk3: btrunk3, trunk4: btrunk4})
+}
+
+// configInterfaceDUT configures bundle members.
+func configInterfaceDUT(i *oc.Interface, a *attrs.Attributes, dut *ondatra.DUTDevice) *oc.Interface {
+	i.Type = oc.IETFInterfaces_InterfaceType_ieee8023adLag
+	if deviations.InterfaceEnabled(dut) {
+		i.Enabled = ygot.Bool(true)
+	}
+	s := i.GetOrCreateSubinterface(0)
+	s4 := s.GetOrCreateIpv4()
+	s4a := s4.GetOrCreateAddress(a.IPv4)
+	s4a.PrefixLength = ygot.Uint8(ipv4PrefixLen)
+	if deviations.InterfaceEnabled(dut) && !deviations.IPv4MissingEnabled(dut) {
+		s4.Enabled = ygot.Bool(true)
+	}
+	return i
+}
+
+// configureBundle configures bundle interfaces.
+func configureBundle(t *testing.T, name, bundleID string) *oc.Interface {
+	i := &oc.Interface{Name: ygot.String(name)}
+	i.Type = oc.IETFInterfaces_InterfaceType_ethernetCsmacd
+	e := i.GetOrCreateEthernet()
+	e.AggregateId = ygot.String(bundleID)
+	return i
+}
+
+// generateIPAddress generates ipv4 addresses.
+func generateIPAddress(dstP string, i int) string {
+	ip := net.ParseIP(dstP)
+	ip = ip.To4()
+	ip[3] = ip[3] + byte(i%256)
+	ip[2] = ip[2] + byte(i/256)
+	ip[1] = ip[1] + byte(i/(256*256))
+	return ip.String()
+}
+
+// addStaticRoute configures static routes.
+func addStaticRoute(t *testing.T, dut *ondatra.DUTDevice) {
+	d := gnmi.OC()
+	s := &oc.Root{}
+	static := s.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut)).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, deviations.StaticProtocolName(dut))
+	ipv4Nh := static.GetOrCreateStatic(innerDst).GetOrCreateNextHop("0")
```

Comment:
Please could you add default network instance type to DEFAULT .

Reply:
This is for the testImplementDrain case (case2) which runs only for Cisco now, as static arp/route deviations needed for other vendors. Also, deviations.DefaultNetworkInstance is "DEFAULT".